defaults/imagedigests.go

```diff
@@ -0,0 +1,24 @@
+package defaults
+
+import (
+	_ "embed"
+	"encoding/json"
+)
+
+//go:embed imagedigests.json
```

Comment:
Doesn't need to be part of this PR, but maybe an idea for a follow-up: Would be nice to have a script/Go program which could be used in a `//go:generate` directive to update `imagedigest.json` to the latest versions/digests per release branch.

Reply:
I've added a manual command to update this now. For `//go:generate` we need a method to find new versions.